8.x/theme-development/static-content.md

```diff
@@ -2,4 +2,212 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+This section explains how to create templates with
+Static Content that can be edited through the admin panel.
+
+For example, the image below shows a page of a website created with Larammerce.
+![example-template.png](/example-template.png)
+
+The contents of this page are text and photos. In the previous section, creating this content was taught statically. But for the site admin to be able to edit this content, you need to get acquainted with the **content** module.
+
+## Content module
+There is a content module in the Larammerce Template Enginee so that the admin can edit the site content. To edit the contents of the html tag, the following three attributes must be added to the tag.
+```bash
+hct-content="variable name"  # Here is added a variable name.
+hct-title='title' # Here is a title that is displayed on the site
```

Comment:
# Here is the title of this section shown to the website admin to modify and change the contents.